requirements-airflow.txt

```diff
@@ -1,5 +1 @@
-apache-airflow[celery]==2.6.0 --constraint https://raw.githubusercontent.com/apache/airflow/constraints-2.6.0/constraints-3.8.txt
-apache-airflow-providers-celery==2.1.3
-celery==5.2.6
-redis==4.2.2
-psycopg2==2.9.6
+apache-airflow[celery,postgres,redis]==2.6.0 --constraint https://raw.githubusercontent.com/apache/airflow/constraints-2.6.0/constraints-3.10.txt
```

Comment:
do we need to keep the requirements files since we move to poerty?

Reply:
no, we don't